base/linalg/generic.jl

```diff
@@ -957,7 +957,7 @@ true
 function istriu(A::AbstractMatrix)
     m, n = size(A)
     for j = 1:min(n,m-1), i = j+1:m
-        if A[i,j] != 0
+        if A[i,j] != zero(eltype(A))
```

Comment:
Its new for the upcoming 0.6 release (https://github.com/JuliaLang/julia/pull/19950) so it is only in the latest docs: https://docs.julialang.org/en/latest/stdlib/numbers.html#Base.iszero